src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java

```diff
@@ -285,6 +285,123 @@ public static List<OcrDataField> getA60OrC63orA58Data() {
         return fieldList;
     }
 
+    public static List<OcrDataField> getFgm001Data() {
+        List<OcrDataField> fieldList = new ArrayList<>();
+        OcrDataField ocrDataFirstNameField = new OcrDataField();
```

Comment:
i see same value is used everywhere in the file, so it's better to create constants and use it.

Reply:
I moved my changes into comments.